docs/developing.rst

```diff
@@ -47,6 +47,57 @@ of Tron create a working directory and start
 
 A sample testing config file is available at ``tests/data/test_config.yaml``
 
+Running Tron under Vagrant
+--------------------------
+
+A Vagrantfile is present that will fire up a working multi-node Tron playground
+environment.
+
+Fire this up with::
+
+    $ vagrant up
+    $ vagrant ssh master.tron-v
+
+:command:`trond` will be running, with no nodes configured.
+
+A basic environment can then be loaded with :command:`tronfig`::
```

Comment:
It would be a good idea to put this config on the master to get everything ready.